rb/lib/selenium/webdriver/common/element.rb

```diff
@@ -29,15 +29,16 @@ class Element
       #
 
       def initialize(bridge, id)
```

Comment:
Oh cool, didn't realize you could do that. I do think that's an unnecessary/confusing abstraction. The Class signature is not the place I would think to look to figure out what the ordered parameters are.